frappe/public/js/frappe/list/list_view.js

```diff
@@ -1320,6 +1320,11 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 	}
 
 	setup_realtime_updates() {
+		this.pending_document_refreshes = [];
+		setInterval(() => {
+			this.process_document_refreshes();
+		}, 1000);
```

Comment:
`Debounce` instead of `setInterval`? wait only if situation multiple update at once happens... also remove a continuous process.

```suggestion
		frappe.utils.debounce(this.process_document_refreshes.bind(this), 500)();
```
also, 500ms should be enough (or even less)?

Reply:
> `Debounce` instead of `setInterval`? wait only if situation multiple update at once happens... also remove a continuous process.

I like `debounce` better, but I think 1000ms is fine.

The 1s interval I added makes 2-3 requests. Even a 1s delay doesn't make any noticeable difference. 500ms means more frequent requests and re-rendering the list view and can cause a clunky experience, the opposite of what we're trying to achieve.